crates/nu-command/src/filters/length.rs

```diff
@@ -64,6 +72,12 @@ fn length_row(call: &Call, input: PipelineData) -> Result<PipelineData, ShellErr
                 src_span: span,
             })
         }
+        PipelineData::Value(Value::Binary { val, .. }, ..) => {
+            Ok(Value::int(val.len() as i64, call.head).into_pipeline_data())
+        }
+        PipelineData::ByteStream(s, _) => {
+            Ok(Value::int(s.into_bytes()?.len() as i64, call.head).into_pipeline_data())
```

Comment:
Note that `ByteStream`s can have type string, binary, or unknown, see:

https://github.com/nushell/nushell/blob/1c3ff179bc575a022b6fa71dfce190c522b705ce/crates/nu-protocol/srhc/pipeline/byte_stream.rs#L83

So, this section of code should check that the byte stream is type unknown or binary (and not string).

Also, it would be nice to get the length of byte stream without having collecting the whole stream into memory (`into_bytes` collects into a `Vec`).